app/routes/__index/domains/new.tsx

```diff
@@ -48,7 +48,7 @@ export const action = async ({ request }: ActionArgs) => {
 
   // Create a pending record... for now
```

Comment:
This comment probably needs to get updated to reflect the new reality.

Reply:
I think the comment is still valid. I just move the status set from here to `/models/record.server.ts`